Goddess JOI

wwilcox999 opened the pull request:
Edited some wording and phrases to sound more like the Goddess.

HeadphoneTaco commented:
Going to tweak animations more, now that I completed the scene map

wwilcox999 committed:
Goddess JOI

Edited some wording and phrases to sound more like the Goddess.

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -779,20 +783,20 @@ Nod your head when you are finished.
     {shuffle:
     - Such a big {player_name}!
-    - Watching my slaves cum always gets me so fucking wet.
+    - The scene of a slaves ejaculation always warms my nether regions.
```
I would say 'the sight'... "the scene" doesn't sound quite right.

wwilcox999 replied:
I agree. Great idea!

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -779,20 +783,20 @@ Nod your head when you are finished.
-    - What a big load from such a small cock.
-    - Mmmm... impressive
-    - I'm almost tempted to go over there and clean you up myself... Maybe one day.
+    - Such an impreesive load from one so puny.
```
typo - impressive

wwilcox999 replied:
Great catch. Sorry about that!!!

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -779,20 +783,20 @@ Nod your head when you are finished.
-    - I'm almost tempted to go over there and clean you up myself... Maybe one day.
+    - Such an impreesive load from one so puny.
+    - Mmmm... impressive.
+    - I'm almost tempted to go over there and clean you up myself... Perhaps one day.
```
Nice catch to tweak the single word to sound more like her.

wwilcox999 replied:
Glad you noticed!

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -779,20 +783,20 @@ Nod your head when you are finished.
-   - I want you to shoot that load into your hand and lick it all up when I order you to cum.
-   - Now go get a spoon to fill with your hot load when I tell you to cum.
+   - Fetch a small vessel. You will fill it when I order you to cum.
+   - I want you to shoot your load into the palm of your hand and lick it all up after I order you to cum.
```
I think the word 'shoot' isn't quite like her... so perhaps it would read better as "I want you to spew your seed into your hand and lick it all up after I order you to cum"

You could switch to 'the palm of...', if you want. I don't think you have to.

wwilcox999 replied:
I agree shoot doesn't sound like her. Great catch :)

wwilcox999 committed:
JOI/"CEI" Vagina

Added JOI and "CEI" for players with a vagina. And changed the wording of a few things.

wwilcox999 committed:
Added more comments/ fixed some dialogue

wwilcox999 committed:
Added more lines and comments. Fixed the wording on some

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -110,7 +110,7 @@ Would you like to wear a gag for me, {player_name}?
 + [yes]
     Excellent.
-    Then go get it.
+    Then go retrieve it.
```
Nice adjustment of language.

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -131,7 +131,7 @@ Would you like to wear a gag for me, {player_name}?
 
 Welcome back, {player_name}.
-I want to see what you have picked out for me.
+Let me gaze upon your choice of plug.
```
I see you're catching me in making sure I embody the Goddess well enough ;)

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -374,7 +374,7 @@ Now, take it nice and slow.
     -> random_comments ->
     
-    - I want you to show me 20 excrutiatingly slow {hasPenis:strokes|circles}.
+    - I want you to show me 20 excrutiating, slow {hasPenis:strokes|circles}.
```
The sentence doesn't quite read right.

It should say, "show me 20 excrutiatingly slow" or "excrutiating and slow"

Alternatively, it's a big word and I'm wondering if "show me 20 painfully slow" would be a little better?

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -436,7 +436,7 @@ I want you to stay at a nice, steady pace.
     - Count the rythm in your head. 
```
I think I mispelled a word! Rhythm.

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -524,7 +524,8 @@ Aren't you?
     - I can see that you're eager to cum for me. 
     - You're pent up with the need to release.
-    - You are desperate to cum for me. 
+    - You are desperate to cum for me. \
```
An extra \ at the end

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -561,6 +562,7 @@ Tell me, {player_name}...
     - Do you enjoy eating your own cum?
     - Would you like to eat your cum for me?
     - Does the thought of eating your own cum excite you?
+    - Does the taste of your own cum fill you with a rush of tingling pleasure?
```
Nice!

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -825,21 +840,26 @@ Nod your head when you are finished.
     - I wonder how you taste.
       Perhaps I'll sample you someday. 
-    - Show me how much you adore me. 
+    - Show me how much you adore me.
```
What should they do to show her?

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -345,7 +345,7 @@ Can you keep continue without orgasming?
     - -> slow ->
     - -> fast ->
-    - -> squeeze ->
+    - {hasPenis: ->squeeze->|->->}
```
Will you write what happens when a person is told to squeeze their lips? Or would you prefer I add that? :)

wwilcox999 committed:
Fixed grammar/ spelling mistakes

wwilcox999 committed:
Added =squeeze(vagina) and =lube

Added the sqeeze option for players with vaginas, and added a lubrication option for an initial variable (like the plug and gag)

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -126,6 +127,25 @@ Would you like to wear a gag for me, {player_name}?
 
 + [distracted] -> Goddess_PayAttention -> mouth
 
+= lube
```
Oooo! Very inspired. I like it!

Don't forget the [timeout] and [distracted] redirects need to go back to 'lube'

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -200,6 +220,26 @@ Your lips bulge beautifully around it.
+/wait3 
+
+{shuffle:
+    - That's it, make yourself nice and messy for me.
```
That's it. Make yourself nice and messy for me.

Period, not comma :)

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -410,6 +450,20 @@ Start going a bit quicker.
 
 - ->->
 
+= squeeze_vagina
```
Very nice!! I like the creative options you came up with.

wwilcox999 committed:
More language/ grammar tweaks

Final grammar tweak.

wwilcox999 committed:
Edited some lines to improve flow of the scene

DevilishDomina commented on ink/goddess/reward_joi:
```diff
@@ -881,7 +881,7 @@ Nod your head when you are finished.
     - Mmmm.
     - So good. 
-    - What a good little slut.
+    - What a good little {player_name}.
```
awesome
